packages/fx-core/resource/package.nls.json

```diff
@@ -257,8 +257,8 @@
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.mitigation": "Mitigation: update \"%s\" field in \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate": "Missing \"%s\" in command \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate.mitigation": " Mitigation: create Adaptive Card template in \"%s\" and then update \"%s\" field to the relative path in \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters": "Missing parameters in command \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters.mitigation": " Mitigation: add at least one parameter for command \"%s\" in \"%s\". The parameter name should match what's defined in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly": "There is no required parameter defined in API \"%s\", so we set the first optional parameter as the parameter for command \"%s\".",
```

Comment:
recommend:
There is no required parameter defined in API \"%s\". The first optional parameter is set as the parameter for command \"%s\".

Reply:
Thanks, updated.